hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestAlterTableDropPartition.scala

```diff
@@ -80,111 +80,92 @@ class TestAlterTableDropPartition extends HoodieSparkSqlTestBase {
     checkAnswer(s"show partitions $tableName")(Seq.empty: _*)
   }
 
```

Comment:
For SQL testing, the valid config is (encodePartition=true, keyGen=SqlKeyGenerator). This PR simplify as below.